tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/LogstashInstallation.java

```diff
@@ -161,7 +166,7 @@ private static void pipeRepeatedly(final File input, final OutputStream out,
             final int count) throws IOException {
             for (int i = 0; i < count; ++i) {
                 try (final InputStream file = new FileInputStream(input)) {
-                    IOUtils.copy(file, out, 16 * 4096);
+                    IOUtils.copy(file, out, 4096 * 16);
```

Comment:
@jakelandis haha not that I know of, umm I admit I don't even remember making that change ... reverting :D